<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="3" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="RetryFailedTestSuite" duration-ms="70" started-at="2018-10-25T11:07:57Z" finished-at="2018-10-25T11:07:57Z">
    <groups>
    </groups>
    <test name="RetryMulitpleFailedTestCases" duration-ms="70" started-at="2018-10-25T11:07:57Z" finished-at="2018-10-25T11:07:57Z">
      <class name="mytests.MyTests">
        <test-method status="PASS" signature="Test1()[pri:0, instance:mytests.MyTests@1810399e]" name="Test1" duration-ms="15" started-at="2018-10-25T11:07:57Z" finished-at="2018-10-25T11:07:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
        <test-method status="PASS" signature="Test2()[pri:0, instance:mytests.MyTests@1810399e]" name="Test2" duration-ms="2" started-at="2018-10-25T11:07:57Z" finished-at="2018-10-25T11:07:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
        <test-method status="PASS" signature="Test3()[pri:0, instance:mytests.MyTests@1810399e]" name="Test3" duration-ms="6" started-at="2018-10-25T11:07:57Z" finished-at="2018-10-25T11:07:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test3 -->
      </class> <!-- mytests.MyTests -->
    </test> <!-- RetryMulitpleFailedTestCases -->
  </suite> <!-- RetryFailedTestSuite -->
</testng-results>
